src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int sideLength;
+
+    public Square(Colors color, int uniLength) {
```

Comment:
```suggestion
    public Square(Colors color, int sideLength) {
```
same with other constructors

Reply:
done